test/index.js

```diff
@@ -26,8 +26,11 @@ describe('default resolution', () => {
       subscription: new GraphQLObjectType({
         name: 'Subscription',
         fields: {
-          foo: subscriptionWithClientId({
+          foo: subscription({
             name: 'FooSubscription',
+            inputFields: {
+              dummy: { type: GraphQLString },
```

Comment:
Empty objects aren't allowed in GraphQL object types, so dropping `clientSubscriptionId` means that users have to specify a dummy input field for these cases.